### PR TITLE
MBL-1858: Payment Schedule component

### PR DESCRIPTION
**Pull request by jlplks**
# 📲 What
Crea te a Payment Schedule component on compose with tests

# 🤔 Why

To show the payment schedule on 'Your pledge section'

# 🛠 How

Created a compose class

# 👀 See

Screenshot_20241126_224813
Screenshot_20241126_224823

### CODE REVIEW

File: app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            onExpandChange = { isExpanded = it }
+        )
+    }
+}
+
+@Composable
+fun PaymentSchedule(
+    isExpanded: Boolean = false,
+    onExpandChange: (Boolean) -> Unit = {},
+    // TODO: Add payment schedule data model when available
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt:93]: fyi, @jlplks we do create our own data models for the app. Once the API layer is ready (either GraphQL or REST) once the network call is made we will make the transformation required within the networking layer, take a look for example at  `KSApolloclient -> override fun getProject(slug: String): Observable<Project>`. The project returned is our own project -> `Project.kt`

**Review by leighdouglas — APPROVED** [review]
lgtm!

File: app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt (hunk 1 above)
Comment (Arkariang) [reply app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt:93]: another fyi, if you want to know how the PaymentIncrement data model is gona look like on the API, take a look here -> https://kickstarter.atlassian.net/wiki/spaces/PAYM/pages/3210641443/PLOT+Mobile+API+Source+of+Truth?focusedCommentId=3212476418

Comment (jlplks) [reply app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt]: Created data model for it

**Review by Arkariang — APPROVED** [review]

**Pull request merged**